src/lib/theme-selector/themes/minimal-runtime.css

```
@@ -139,11 +275,26 @@
   --color-fg-indigo: var(--color-indigo-500);
-  --color-fg-pink: var(--color-pink-500);
   --color-fg-lime: var(--color-lime-500);
+  --color-fg-orange: var(--color-orange-500);
+  --color-fg-pink: var(--color-pink-500);
+  --color-fg-purple: var(--color-purple-500);
+  --color-fg-red: var(--color-red-500);
+  --color-fg-rose: var(--color-rose-500);
+  --color-fg-sky: var(--color-sky-500);
+  --color-fg-teal: var(--color-teal-500);
+  --color-fg-violet: var(--color-violet-500);
+  --color-fg-yellow: var(--color-yellow-500);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--color-fg-yellow is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)